mozilla/fx-private-relay
# (draft) new API endpoints

# API endpoit to create outbound calls
# TODOs
1. [ ] Put this behind waffle flag
2. [ ] Check the relay number has remaining call minutes
3. [ ] After the call, deduct the number of minutes of the call

## Screenshot (if applicable)
![image](https://user-images.githubusercontent.com/71928/201347689-c9a554c6-3288-4d3c-9f3d-fad4375f4455.png)

## How to test
1. Sign in to http://127.0.0.1:8000 with an account that has phone level of Relay
4. Go to http://127.0.0.1:8000/api/v1/docs/
5. Expand the `POST /call` endpoint at the top
6. Click "Try it out"
7. Replace `{"to": "string"}` with `{"to": "+12223334444"}` (use a real  number you have access to)
8. Click "Execute"
   * [ ] You should receive a call to your true phone number from your relay number
   * [ ] When you answer, it should say "Dialing +12223334444"
   * [ ] It should connect your call to +12223334444


# Checklist
